### ISSUE TITLE
Include loss from training data

### ISSUE THREAD

**Issue by beckynevin** [deepskies/DeepUQ#107]
in the checkpoint so you can overplot on the loss plots and see if its smoother
